utils/rpms/daos.spec

```
@@ -407,6 +416,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 %{_prefix}/lib/daos/.build_vars.json
 %{_prefix}/lib/daos/.build_vars.sh
 
+%files tests-java
+%dir %{_prefix}//lib/daos/TESTING/ftest/java/
```

You want everything under `%{_prefix}//lib/daos/TESTING/ftest/java/`, not just the `%{_prefix}//lib/daos/TESTING/ftest/java/` itself:
```suggestion
%{_prefix}//lib/daos/TESTING/ftest/java/
```